weave-js/src/components/BytesStoredInfoIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import numeral from 'numeral';
+import React from 'react';
+
+import {IconInfo} from './Icon';
+import {Tailwind} from './Tailwind';
+import {Tooltip as WeaveTooltip} from './Tooltip';
+
+export const BytesStoredInfoIcon = ({bytesStored}: {bytesStored: number}) => (
+  <Tailwind>
+    <div className="flex items-center">
+      {numeral(bytesStored).format('0.0b')}
+      <WeaveTooltip
+        content="This does not take into account any files or media logged in this trace."
```

Comment:
This message's reference to "this trace" is confusing when used with an object version.

Reply:
That's a great callout! Added another prop `weaveKind` to specify there :) 
<img width="652" alt="Screenshot 2024-10-24 at 12 11 53 PM" src="https://github.com/user-attachments/assets/4d9db5c4-b25a-40a2-be65-b88af67ebf6e">
<img width="628" alt="Screenshot 2024-10-24 at 12 11 43 PM" src="https://github.com/user-attachments/assets/b1fb5d3e-cb21-44aa-be6e-842563b31ff0">